core/kernel/console.c

```diff
@@ -20,6 +21,11 @@ static struct serial_chip *serial_console __nex_bss;
 
 void console_init(void)
 {
+	if (IS_ENABLED(CFG_SEMIHOSTING_CONSOLE)) {
+		semihosting_console_init(CFG_SEMIHOSTING_CONSOLE_FILE);
```

Comment:
Please add these two CFG_ to `mk/config.mk` (i.e.e, move what you have in the last commit into "drivers: Implement semihosting based console driver for log").

Reply:
Done https://github.com/OP-TEE/optee_os/pull/6700/commits/f542f6c9e8bb12571ee922bbb715b698c084ee4c#diff-6301a2703b709daa4d84ead1c30bf67cdaf088abed68753a8f9c740f1e58b54a